core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -380,6 +380,10 @@ private static int mkGeoHashType(int bits, short baseType) {
                 }
             }
         }
```

Comment:
Something to keep in mind: 
This change has a bit weird-ish side-effect of treating null as string even though it could be mapped to e.g. int . 
It's visible in ExplainPlanTest below where instead of EqIntFunctionFactory, compiler decides to use EqIntStrCFunctionFactory. It's still fine because null str is mapped  to null int .